frontend/src/components/happening-meta-info.tsx

```diff
@@ -21,6 +23,8 @@ interface Props {
     companyLink: string | null;
     spotRangeCounts: Array<SpotRangeCount> | null;
     spotRangesFromCms: Array<SpotRange> | null;
+    deductible: boolean;
```

Comment:
Det er vel litt overflødig og ta med `deductible` i typen her? Det er kanskje greit å ha den i schema til Sanity (@boaanes?), men at `deductible` er `true` er egentlig bare det samme som at `deductiblePayment` ikke er `null`.

Reply:
Planen var at deductibles skal være en avkrysningsboks, og så dukker feltet for å skrive inn beløp opp dersom man huker av for at det skal være egenandel. @boaanes jobber med en slik funksjonalitet i Sanity slik jeg forstod det? Men er kanskje overflødig med begge i mellomtiden, ja

Reply:
@elefantoft @bakseter Den er litt overflødig å hente ned til frontenden ja. Den er fin å ha med i schema i cms for å gjøre brukeropplevelsen i sanity fin, men som sagt, ikke vits å hente ut i spørringen.